mungegithub/mungers/close-stale.go

```diff
@@ -81,17 +81,7 @@ func (CloseStale) EachLoop() error { return nil }
 // AddFlags will add any request flags to the cobra `cmd`
 func (CloseStale) AddFlags(cmd *cobra.Command, config *github.Config) {}
 
-func findLastHumanPullRequestUpdate(obj *github.MungeObject) (*time.Time, bool) {
-	pr, ok := obj.GetPR()
-	if !ok {
-		return nil, ok
-	}
-
-	comments, ok := obj.ListReviewComments()
-	if !ok {
-		return nil, ok
-	}
-
+func findLastHumanPullRequestUpdateHelper(pr *githubapi.PullRequest, comments []*githubapi.PullRequestComment) (*time.Time, bool) {
```

Comment:
You probably don't need the PR here. It's used only for `CreatedAt`, which is also in the issue, and should probably be passed as a parameter.